kore/src/Kore/Log/DebugRewriteTrace.hs

```diff
@@ -131,7 +133,7 @@ instance ToJSON DebugInitialClaim where
     toJSON (DebugInitialClaim uniqueId claim) =
         object
             [ "task" .= ("reachability" :: Text)
-            , "claim" .= unparseOneLine claim
+            , "claim" .= (unSource . source) claim
```

Comment:
Can you instead use the `Pretty` instance here?

Reply:
Changed. Result doesn't look better: path to source is just accompanied with source position:
```
---
claim-id: null
claim: /home/alexey/Projects/RV/claims/simpl-func.k:7:11-7:86
task: reachability
steps:
```